app/models/katello/content_view_version.rb

```diff
@@ -73,8 +73,8 @@ def self.with_library_repo(repo)
 
     def self.for_version(version)
       major, minor = version.to_s.split('.')
-      query = where(:major => major)
-      query.where(:minor => minor) if minor
```

Comment:
@daviddavis according to the [this line] (https://github.com/Katello/katello/blob/master/test/models/content_view_version_test.rb#L41) of the tests you mentioned, it appears that only 1 content view version should be returned when filtering by "1" or a major version, just 1.0 would be returned like you suggested. There is a [minor version](https://github.com/Katello/katello/blob/master/test/models/content_view_version_test.rb#L8) that would be returned if filtering by "1" returned 1.1 as well. So I think you are correct, only 1.0 should be returned instead of 1.1, 1.2, etc..